Sources/XCTest/XCAbstractTest.swift

```diff
@@ -33,4 +64,8 @@ public class XCTest {
     /// Teardown method called after the invocation of each test method in the
     /// class.
     public func tearDown() {}
+
+    // FIXME: This initializer is required due to a Swift compiler bug on Linux.
+    //        It should be removed once the bug is fixed.
+    public init() {}
```

Comment:
Oh, good point. I have a feeling this is subtly different from SR-1129, but I'll double check.